src/background/actions/details-view-actions.ts

```diff
@@ -8,6 +8,5 @@ export class DetailsViewActions {
         DetailsViewRightContentPanelType
     >();
     public readonly getCurrentState = new Action();
-    public readonly openSettingsPanel = new Action();
```

Comment:
Why do we no longer need an action for opening side panel? 

Also for the `closeSettingsPanel` action, should it be renamed to `closeSidePanel`?

Reply:
Not yet. `closeSettingsPanel` only allows to close that specific panel. As mention in the description, this is the first PR in a series of smaller refactors in order to change how we open/close panel. At this point, I didn't want to change any other action expect for how we open the settings panel.